tables/tables.go

```diff
@@ -223,6 +223,23 @@ var AndroidNativeRules = []string{
 // AndroidLoadPath is the load path for the Starlark Android Rules.
 var AndroidLoadPath = "@rules_android//android:rules.bzl"
 
+// JavaNativeRules lists all Java rules that are being migrated from Native to Starlark.
+var JavaNativeRules = []string{
+	"java_binary",
+	"java_import",
+	"java_library",
+	"java_lite_proto_library",
+	"java_proto_library",
+	"java_test",
+	"java_package_configuration",
+	"java_plugin",
+	"java_runtime",
+	"java_toolchain",
+}
+
+// AndroidLoadPath is the load path for the Starlark Android Rules.
```

Comment:
Please fix the comment (golint will warn about it).

Reply:
Oh cool. Thanks for catching it. Done.